lib/application.ts

```diff
@@ -811,10 +812,18 @@ export class SNApplication {
        * stop importing if there is no backup file version, only if there is
        * an unsupported version.
        */
+      const version = data.version as ProtocolVersion;
+
       const supportedVersions = this.protocolService!.supportedVersions();
-      if (!supportedVersions.includes(data.version as ProtocolVersion)) {
+      if (!supportedVersions.includes(version)) {
         return { error: UNSUPPORTED_BACKUP_FILE_VERSION };
       }
+
+      const userVersion = await this.getUserVersion();
+      if (userVersion && compareVersions(version, userVersion) === 1) {
```

Comment:
Same check here, version can be undefined.

Reply:
https://github.com/standardnotes/snjs/pull/107/files/7b9c188e7fac16ea14f7018f9bc1cd22f3ab9579#diff-958a601413ee4d47cb1165fe9ed67ffcL808